JsonFile.js

```diff
@@ -2,8 +2,12 @@
 
 let fsp = require('mz/fs');
 let _ = require('lodash');
+let path = require('path');
 let util = require('util');
 let JSON5 = require('json5');
+let writeFileAtomic = require('write-file-atomic');
+var lockFile = require('lockfile');
+require('instapromise');
```

Comment:
It's enabled only on Node 8 but we generally support 6 and up (sometimes 7 and up)